src/prometheus/deploy/refresh.sh

```diff
@@ -20,9 +20,7 @@
 
 pushd $(dirname "$0") > /dev/null
 
-
-echo "refresh prometheus configuration"
-kubectl apply -f prometheus-configmap.yaml || exit $?
```

Comment:
I'm wandering why it doesn't work. Do you have any idea?

Reply:
this only works if someone changed some parameters that are allowed to change in description of configmap. It will not restart prometheus service, which might be expected behavior by refresh user.